tools/download_python_package.py

```diff
@@ -0,0 +1,125 @@
+import os
+import subprocess
+import time
+import shutil
+import sys
+import argparse
+
+VERSION = "1.0.0"
+
+__version__ = VERSION
+__author__ = "Chris Cianelli"
+
+
+def download_python_package(
+    python_version: str, module_name: str, module_version: str, host_directory: str
+):
+    """
+    Download a Python package and its dependencies for local use
+
+    Parameters
+    ----------
+    python_version: str
+        Python version to use. Ex: "3.8.5" or "3.9"
+    module_name: str
```

Comment:
bit of a nitpick but these are usually called package/package version - "module" usually refers to an invidividual python file.

Reply:
updated